test/services/AuditServiceSpec.scala

```diff
@@ -49,7 +49,8 @@ class AuditServiceSpec extends UnitSpec with Matchers with MockStorageService wi
       deskProServiceId = "vrs",
       signOutUrl = testSignOutUrl
     ),
-    LimitedCompany
+    LimitedCompany,
+    businessVerificationCheck = testBusinessVerificationCheck
```

Comment:
I think it would read easier if this just said true instead of being a test variable in a different file